docs/content/concepts/assets/asset-checks.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: Asset checks | Dagster
+description: Asset checks are a way to define expectations about the quality of your data.
+---
+
+# Asset checks (Experimental)
+
+Dagster allows you to define and execute data quality checks on your [software-defined assets](/concepts/assets/software-defined-assets). Each asset check verifies some property of a data asset, e.g. that is has no null values in a particular column.
+
+When viewing an asset in Dagster’s UI, you can see all of its checks, and whether they’ve passed, failed, or haven’t run.
+
+## Defining asset checks
+
+### Asset check that executes in its own op
+
+These are the bare APIs for defining an asset and an asset check that executes in its own op.
+
+```python file=/concepts/assets/asset_checks/simple_check.py
+from dagster import AssetCheckResult, Definitions, asset, asset_check
+
+
+@asset
+def my_asset():
+    ...
+
+
+@asset_check(asset=my_asset)
+def my_check(context):
+    return AssetCheckResult(success=True)
+
+
+defs = Definitions(assets=[my_asset], asset_checks=[my_check])
+```
+
+[`@asset_check`](/\_apidocs/asset-checks#dagster.asset_check) is a decorator for a function that returns a [`AssetCheckResult`](/\_apidocs/asset-checks#dagster.AssetCheckResult) object.
```

Comment:
I believe we prefer to use `<PyObject...>` for these links.  You can grep and see other examples in the docs.